src/static/current-state/components/tabs/SystemTab.tsx

```diff
@@ -115,6 +139,27 @@ export class SystemTab extends React.Component<SystemTabProps, SystemTabsState>
                 />
                 <label htmlFor="pause-toggle">Option</label>
               </div>
+              <div
+                className="ak-field-toggle ak-field-toggle__size-large"
+                style={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                  fontSize: '14px',
+                  marginTop: '10px',
+                }}
+              >
+                <span>Merge blocking: </span>
+                <input
+                  type="checkbox"
+                  name="merge-blocking-toggle"
+                  id="merge-blocking-toggle"
+                  value="merge-blocking-toggle"
+                  checked={this.state.adminSettings.mergeBlockingEnabled}
+                  onChange={this.handleMergeBlockingChange}
+                />
+                <label htmlFor="merge-blocking-toggle">Merge blocking enabled</label>
```

Comment:
It would be good to hide this UI completely if the config is disabled. Since toggling the setting won't achieve anything if there isn't config and it'd be showing unnecessary UI to users that don't have this enabled in config.

nit: Could be a good idea to add a tooltip of some sort that explains the merge blocking feature in more detail.